app/routes/articles/components/Overview.tsx

```diff
@@ -24,17 +24,21 @@ const OverviewItem = ({ article }: { article: ArticleWithAuthorDetails }) => (
     </h2>
 
     <span className={styles.itemInfo}>
-      {article.author.id && (
-        <span>
-          <Link
-            to={`/users/${article.author.username}`}
-            className={styles.overviewAuthor}
-          >
-            {' '}
-            {article.author.fullName}
-          </Link>{' '}
-        </span>
-      )}
+      {article?.author &&
+        article?.author?.map((e) => {
```

Comment:
Here as well, when you use the `?.` operator, you don't need the `article?.author &&` check.